test/eigen.jl

```diff
@@ -323,4 +323,10 @@ using StaticArrays, Test, LinearAlgebra
             @test vecs[:,1] ≈ conj.(vecs[:,2])
         end
     end
+
+    @testset "non-float matrix" begin
+        A = SMatrix{3,3}(1:9)
+        λ, V = eigen(A)
```

Comment:
```suggestion
        λ, V = eigen(Symmetric(A))
```
The non-symmetric case already works.

Reply:
Thanks, I've updated the code to implement the suggestion